### ISSUE TITLE
Oven recipe is unobtainable.

### ISSUE THREAD

**Issue by blot-runner** [cataclysmbnteam/Cataclysm-BN#5770]
### Describe the bug

As by the title, the oven recipe lacks both the books it can be learned from, as well as ""autolearned": true" value, making it unobtainable in the game.

### Steps To Reproduce

1. Create a new world, no mods.
2. Create a new character.
3. Debug yourself Fabrication 5+, open craft window, search for "oven". Observe no entries are found.
4. Press 'U' (default keybinding) to show all craftable items. Observe the oven recipe is present, but marked "Not known" and lacking any books the recipe can be learned from (book count: 0).
ALTERNATIVELY: Check data\json\recipes\other\other.json file for the recipe, observe no "autolearn" and no "book_learn" values set (thus autolearn defaults to false).

Expecting to either be able to autolearn the recipe, learn it from a book or else the recipe being absent entirely. My guess it should be autolearned.  

### Screenshots

![image](https://github.com/user-attachments/assets/0e220581-24bb-4059-9a47-1ef492e6fbc5)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.5131 (22H2)
- Game Version: 23deeac [64-bit]
- Graphics Version: Tiles
- LAPI Version: 2
- Game Language:  []
- Mods loaded: [
    Bright Nights [bn]
]

**Comment by ZChris13** [comment]
can't you still disassemble ovens for it? My current character has the recipe, so that's how he must have gotten it

**Comment by Proxiehunter** [comment]
> can't you still disassemble ovens for it? My current character has the recipe, so that's how he must have gotten it

Maybe, but that shouldn't be the _only_ way. Among other reasons, if you already _have_ an oven to disassemble to learn the recipe (or more than one if you don't learn it first try) then you _already_ have an oven and you can just take that one back to your base. Taking one apart and reassembling it seems like a lot of effort to go to just so you don't have to wield and oven and waddle back to your base (assuming you don't have a vehicle you can put it in for the trip). All the components are going to weigh the same as just wielding it anyway.

**Comment by chaosvolt** [comment]
Fridges can be learned, right? If so then definitely valid to allow adding a recipe for an oven.

**Comment by Treah** [comment]
Yes chaosvolt the fridge is learned in several books only. However the oven has no such tag so its learn by disassembly only. This is a simple json change so If there is no other person who submits a PR then I can.  I can just use the same textbooks that the fridge uses if everyone is cool with that?

**Comment by chaosvolt** [comment]
Should be fine to use the same books, ye.

**Comment by chaosvolt** [comment]
Fixed by https://github.com/cataclysmbnteam/Cataclysm-BN/pull/5780
